src/lib/net/address.c

```diff
@@ -903,6 +903,20 @@ tor_addr_from_in6(tor_addr_t *dest, const struct in6_addr *in6)
   tor_addr_from_ipv6_bytes(dest, (const char*)in6->s6_addr);
 }
 
+/** Set the 16 bytes at <b>dest</b> to equal the IPv6 address <b>src</b>.
+ * <b>src</b> must be an IPv6 address, if it is not, log a warning, and clear
+ * <b>dest</b>. */
+void
+tor_addr_get_ipv6_bytes(char *dest, const tor_addr_t *src)
```

Comment:
This should use uint8_t, not char for the output.

Would "copy" make more sense than "get" for the function name?

Reply:
I matched tor_addr_from_ipv6_bytes(), but I'll change them both to uint8_t.

I can also change the function name.

Reply:
There were only 3 address.h functions that still used `char *` for bytes, so I fixed them all in cd7e2fc210349615c8e9cdb879f98cb0c9cac57b.

Rename in f62b051e873aa4be564e9f83745902d4541f79f4.